src/resources/boot.lua

```diff
@@ -61,6 +61,9 @@ function lovr.boot()
       thread = true,
       timer = true
     },
+    graphics = {
+      singlepass = true
+    },
     gammacorrect = false,
```

Comment:
At some point I want to move gammacorrect into `graphics`, but it's a breaking change.